src/panels/SkyBrowserPanel/BrowserTabs/SettingsDisplayCopies.tsx

```diff
@@ -0,0 +1,75 @@
+import { Fragment,useState } from 'react';
+import { Button, Group, Title } from '@mantine/core';
+
+import { useOpenSpaceApi } from '@/api/hooks';
+import { Collapsable } from '@/components/Collapsable/Collapsable';
+import { NumericInput } from '@/components/Input/NumericInput/NumericInput';
+import { Property } from '@/components/Property/Property';
+import { ValueList } from '@/components/Property/Types/VectorProperty/ViewOptions/DefaultValueList';
+import { useAppSelector } from '@/redux/hooks';
+
+interface Props {
+  id: string;
+}
+export function SettingsDisplayCopies({ id }: Props) {
+  const [newPosition, setNewPosition] = useState<[number, number, number]>([0, 0, -2]);
+  const [noOfCopies, setNoOfCopies] = useState(1);
+  const luaApi = useOpenSpaceApi();
+  const displayCopies = useAppSelector(
+    (state) => state.skybrowser.browsers[id]?.displayCopies
+  );
+
+  return (
+    <>
+      <Property uri={`ScreenSpace.${id}.Scale`} />
+      <Property uri={`ScreenSpace.${id}.FaceCamera`} />
+      <Property uri={`ScreenSpace.${id}.UseRadiusAzimuthElevation`} />
+      <Collapsable title={'Add New Display Copy Settings'}>
```

Comment:
```suggestion
      <Collapsable title={<Title order={4}>Add New Display Copy Settings</Title>}>
```
following the other suggested hierarchy